diff --git a/vlmeval/config.py b/vlmeval/config.py
--- a/vlmeval/config.py
+++ b/vlmeval/config.py
@@ -2427,5 +2427,6 @@ lfm2vl_series = {
     "LFM2-VL-450M": partial(vlm.LFM2VL, model_path="LiquidAI/LFM2-VL-450M"),
     "LFM2-VL-1.6B": partial(vlm.LFM2VL, model_path="LiquidAI/LFM2-VL-1.6B"),
     "LFM2-VL-3B": partial(vlm.LFM2VL, model_path="LiquidAI/LFM2-VL-3B"),
+    "LFM2.5-VL-450M": partial(vlm.LFM2VL, model_path="LiquidAI/LFM2.5-VL-450M"),
     "LFM2.5-VL-1.6B": partial(vlm.LFM2VL, model_path="LiquidAI/LFM2.5-VL-1.6B"),
 }
